components/SearchBar/index.js

```diff
@@ -71,9 +74,10 @@ function SearchBar({ search, onSearch }) {
               </Styled.SearchInputSelect>
             </Styled.FilterItem>
             <Styled.FilterItem>
-              <label>Direction</label>
+              <label for="direction">Direction</label>
```

Comment:
[We need to use `htmlFor` in React](https://legacy.reactjs.org/docs/dom-elements.html#htmlfor).